src/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.CoreCLR.cs

```diff
@@ -328,7 +328,7 @@ private static void OnAssemblyLoad(RuntimeAssembly assembly)
 
             foreach (ResolveEventHandler handler in eventHandler.GetInvocationList())
             {
-                Assembly asm = handler(null /* AppDomain */, args);
```

Comment:
All/most events on AppDoman have this problem. Look for `/* AppDomain */`. We should do this to all of them.